test/ParseableInterface/ModuleCache/prebuilt-module-cache-forwarding.swift

```diff
@@ -35,7 +35,7 @@
 // RUN: %empty-directory(%t/MCP)
 // RUN: not %target-swift-frontend -typecheck -parse-stdlib -module-cache-path %t/MCP -sdk %S/Inputs -I %S/Inputs/prebuilt-module-cache/ -prebuilt-module-cache-path %t/prebuilt-cache %s 2>&1
 // RUN: %{python} %S/Inputs/check-is-forwarding-module.py %t/MCP/Lib-*.swiftmodule
-// RUN: chmod a-r %t/MCP/Lib-*.swiftmodule
+// RUN: ls %t/MCP/Lib-*.swiftmodule | %{python} %S/../Inputs/make-unreadable.py
```

Comment:
I don't think this one is doing the same thing. Don't you want `%{python} %S/../Inputs/make-unreadable.py %t/MCP/Lib-*.swiftmodule`?

Reply:
Unfortunately, I don't believe that will actually do the expansion (due to the way that the command line and lit work on Windows).  This should cause the expansion to be passed to the script.

Reply:
It does, but the script doesn't read from stdin.

Reply:
Ah, crap ... need to see if I can use `xargs`

Reply:
Does Windows lit.py really not expand * if it's not going to something that looks shell-ish, like `ls`?

Reply:
The Windows shell does not expand the globs, but lit.py does in some cases.  I'll double check if we can get away with it here.

Reply:
Seems to still pass with glob at the end.